libtransmission/tr-dht.cc

```diff
@@ -46,16 +49,9 @@
 #include "utils.h"
 #include "variant.h"
 
-#define logdbg(...) tr_logAddNamed(TR_LOG_DEBUG, "dht", __VA_ARGS__)
-#define logdbgTor(tor, ...) tr_logAddNamed(TR_LOG_DEBUG, tr_torrentName(tor), __VA_ARGS__)
-#define loginfo(...) tr_logAddNamed(TR_LOG_INFO, "dht", __VA_ARGS__)
-#define logtrace(...) tr_logAddNamed(TR_LOG_TRACE, "dht", __VA_ARGS__)
-#define logtraceTor(tor, ...) tr_logAddNamed(TR_LOG_TRACE, tr_torrentName(tor), __VA_ARGS__)
-#define logwarn(...) tr_logAddNamed(TR_LOG_WARN, "dht", __VA_ARGS__)
-#define logwarnTor(tor, ...) tr_logAddNamed(TR_LOG_WARN, tr_torrentName(tor), __VA_ARGS__)
-
 using namespace std::literals;
 
+static auto constexpr CodeName = "dht"sv;
```

Comment:
I think you were changing from `CodeName` to `LogName` in previous commit...

Reply:
Yes. 5fb750bec095b0138136746ab15c3154a017d22c